src/components/transactions/TxDetails/index.tsx

```diff
@@ -64,10 +64,12 @@ const TxDetailsBlock = ({ txSummary, txDetails }: TxDetailsProps): ReactElement
   const isTrustedTransfer = !hasDefaultTokenlist || isTrustedTx(txSummary)
   const isImitationTransaction = isImitation(txSummary)
 
-  let proposer, safeTxHash
+  let proposer, safeTxHash, proposedByDelegate
   if (isMultisigDetailedExecutionInfo(txDetails.detailedExecutionInfo)) {
     proposer = txDetails.detailedExecutionInfo.proposer?.value
     safeTxHash = txDetails.detailedExecutionInfo.safeTxHash
+    // @ts-expect-error TODO: Need to update the types from the new SDK
+    proposedByDelegate = txDetails.detailedExecutionInfo.proposedByDelegate
```

Comment:
This requires us to change places where we use the old `TransactionDetails` type to the new `components["schemas"]["TransactionDetails"]` type which is a lot of refactor work so I suggest to do this separately.